data/human/news.txt

```diff
@@ -12,7 +12,11 @@
 
 news "human merchant anywhere"
 	location
-		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent" "Hai"
+		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent" "Hai" "Quarg"
+		not
+			government "Quarg"
+			not attributes "human quarg" "hai quarg"
+			
```

Comment:
```suggestion
```

Reply:
What's the problem with this one? If I remember correctly, this is to assure that only human and hai quargs get the human merchant texts, and not the korath ones.

Reply:
This is a whitespace fix.